dev-java/openjdk/openjdk-11.0.23_p9.ebuild

```diff
@@ -302,7 +302,7 @@ src_install() {
 	if use doc ; then
```

Comment:
I suppose this needs a revbump (git mv here) because it might have installed without the symlink.